Cargo.toml

```diff
@@ -46,7 +46,7 @@ winapi = { version = "0.3", features = ["timeapi", "wow64apiset"] }
 
 [dev-dependencies]
 byteorder = "1.3.1"
-rbspy-testdata = "0.1.2"
+rbspy-testdata = { git = "https://github.com/akhramov/rbspy-testdata", branch = "master" }
```

Comment:
I think it would be ok to just point this at rbspy/rbspy-testdata and branch master. I don't think we gain much by having it be versioned and pushed to crates.io?

Reply:
Sure!